sql/core/src/main/scala/org/apache/spark/sql/json/JSONRelation.scala

```diff
@@ -105,6 +107,15 @@ private[sql] class JSONRelation(
     jsonSchema
   }
 
+  override private[sql] def buildScan(
+      requiredColumns: Array[String],
+      filters: Array[Filter],
+      inputPaths: Array[String],
+      broadcastedConf: Broadcast[SerializableConfiguration]): RDD[Row] = {
```

Comment:
No, 'cause we're not using `SqlNewHadoopRDD` here. This method was originally marked as `final` in `HadoopFsRelation` and wasn't supposed to be overriden in concrete data source implementations.